src/transformState.js

```diff
@@ -5,7 +5,30 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
use [destructuring assignment](https://www.geeksforgeeks.org/destructuring-assignment-in-javascript/)

Reply:
i cant use this

Reply:
i cant use this:
const [ action ] = actions;
const { type } = action: 
switch (type) { 
...
because i need loop